datashader/glyphs.py

```diff
@@ -66,3 +67,182 @@ def _compute_x_bounds(self, df):
 
     def _compute_y_bounds(self, df):
         return df[self.y].min(), df[self.y].max()
+
+
+class Line(Glyph):
+    """A point, with points defined by ``x`` and ``y``.
+
```

Comment:
While that's *technically* correct, I feel that most users would call it a line. I'm mostly just mirroring bokeh's plotting interface here: http://bokeh.pydata.org/en/latest/docs/user_guide/plotting.html#single-lines